content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -145,25 +153,27 @@ cases precedence will be given first to the longest matching path. If two paths
 are still equally matched, precedence will be given to paths with an exact path
 type over prefix path type.
 
+## Hostname Wildcards
+Hosts can be precise matches (for example “`foo.bar.com`”) or a wildcard (for example “`*.foo.com`”).
+Precise matches require that the http host header matches the Host setting.
+Wildcard matches require the http host header is equal to the suffix of the wildcard rule.
+
+| Host        | Host header       | Match?                                            |
+| ----------- |-------------------| --------------------------------------------------|
+| `*.foo.com` | `bar.foo.com`     | Matches based on shared suffix                    |
+| `*.foo.com` | `baz.bar.foo.com` | No match, wildcard only covers a single DNS label |
+| `*.foo.com` | `foo.com`         | No match, wildcard only covers a single DNS label |
```

Comment:
This is really helpful, maybe we should have something similar for path type. We could potentially include a few of the examples listed in the KEP here: https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1453-ingress-api/README.md#examples